### DISCUSSION TITLE
[osu!catch] Platter/Catcher does not accurately reflect circle size

### DISCUSSION THREAD

**Discussion by sinklectb** [ppy/osu#24818] [Ideas] [answered]
Maybe rework it so that it works like Stable. As of right now it feels uncomfortable having a small platter trying to catch big fruit

**Comment by peppy** [comment]
which skin are you playing with?

**Reply by sinklectb** [reply]
I tried about 10 or so different skins and it seems that some do scale correctly while others do not. Here are some screenshots showcasing this bug with my main skin. Maybe it has something to do with the way some skins were made. Could just be that the importing process from osu!stable to osu!lazer can by quite finnicky at times. 
![osu_2023-09-15_01-06-03](https://github.com/ppy/osu/assets/145086751/b7a5b35a-4522-40d1-8cd2-87bfdc7edb29) cs 4.3
![osu_2023-09-15_01-06-47](https://github.com/ppy/osu/assets/145086751/fb093948-6dbd-49c6-bac1-4fb6626bd66b) cs 2
![osu_2023-09-15_01-07-20](https://github.com/ppy/osu/assets/145086751/61dd1e2d-d946-4ca0-a8a6-83dfec2b0fee) cs 6

These are just a few examples. Not all skins do this though.

**Reply by sinklectb** [reply]
this may look normal but the platter is increasingly smaller compared to osu!stable

**Reply by peppy** [reply]
If you have a broken skin please upload the skin. This is going to be a per-skin issue, which means that skins might be doing something we don't expect (likely not following the skin template with catcher size, or providing incorrect `@2x` resources or similar.

**Reply by sinklectb** [reply]
Because the file is too large, I am unable to upload the .osk here but I can give a google drive link. https://drive.google.com/file/d/1y5lZzItmyodU78VNPaknJihHV8U-L7Yt/view?pli=1

**Comment by peppy** [comment]
I believe @frenzibyte is tracking this issue (if so please create an issue and self-assign).

**Reply by frenzibyte** [reply, marked as answer]
Tracked already at https://github.com/ppy/osu/issues/18968
